Bio/PDB/MMCIFIO.py

```diff
@@ -0,0 +1,301 @@
+"""Write an mmCIF file.
```

Comment:
Please leave a blank line after the module docstring title, see PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings